src/cmdstan-guide/pathfinder_config.Rmd

```diff
@@ -0,0 +1,180 @@
+posterior.
+
+The full set of configuration options available for the `pathfinder` method is
+reported at the beginning of the pathfinder output file as CSV comments.
+When the example model `bernoulli.stan` is run with `method=pathfinder`
+via the command line with all default arguments,
+the resulting Stan CSV file header comments show the complete set
+of default configuration options:
+```
+method = pathfinder
+  pathfinder
+    init_alpha = 0.001 (Default)
+    tol_obj = 9.9999999999999998e-13 (Default)
+    tol_rel_obj = 10000 (Default)
+    tol_grad = 1e-08 (Default)
+    tol_rel_grad = 10000000 (Default)
+    tol_param = 1e-08 (Default)
+    history_size = 5 (Default)
+    num_psis_draws = 1000 (Default)
+    num_paths = 4 (Default)
+    max_lbfgs_iters = 1000 (Default)
+    num_draws = 1000 (Default)
+    num_elbo_draws = 25 (Default)
+```
+
+The arguments `init_alpha` through `history_size` are used by the L-BFGS algorithm.
+The remaining arguments are used by the Pathfinder algorithm.
+
+## Pathfinder Configuration
+
+- `num_psis_draws` - Number of draws from PSIS sample (multi-path pathfinder).
+Must be a positive integer.  Default value is $1000$.
+
+- `num_paths` - Number of single pathfinders.
+Must be a positive integer.  Default value is $4$.
+
+- `max_lbfgs_iters` - Maximum number of L-BFGS iterations.
+Must be a positive integer.  Default value is $1000$.
+
+- `num_draws` - Number of approximate posterior draws for each single pathfinder.
+Must be a positive integer.  Default value is $1000$.
+
+- `num_elbo_draws` - Number of Monte Carlo draws to evaluate ELBO.
+Must be a positive integer.  Default value is $25$.
+
+
+## L-BFGS Configuration
+
+- `init_alpha` - The initial step size parameter.
+Must be a positive real number. Default value is $0.001$
+
+- `tol_obj` - Convergence tolerance on changes in objective function value.
+Must be a positive real number. Default value is $1^{-12}$.
+
+- `tol_rel_obj` - Convergence tolerance on relative changes in objective function value.
+Must be a positive real number. Default value is $1^{4}$.
+
+- `tol_grad` - Convergence tolerance on the norm of the gradient.
+Must be a positive real number. Default value is $1^{-8}$.
+
+- `tol_rel_grad` - Convergence tolerance on the relative norm of the gradient.
+Must be a positive real number. Default value is $1^{7}$.
+
+- `tol_param` - Convergence tolerance on changes in parameter value.
+Must be a positive real number. Default value is $1^{-8}$.
+
+- `history_size`  - The size of the history it uses to approximate the Hessian.
+The value should be less than the dimensionality of the parameter space and,
+in general, relatively small values ($5$-$10$) are sufficient; the default value is $5$.
+
+## Output CSV files
+
+Pathfinder output is written into a set of StanCSV files; one file which contains the importance resampled draws
+from multi-path Pathfinder, and one file which contains the output of each single-path Pathfinder run.
```

Comment:
We should describe separately what happens if num_paths == 1. I think that we don't append anything to the requested csv name, but if we do we should just note that somewhere

Reply:
added.